uint16 oculusId
uint16 srcDeviceId
uint16 dstDeviceId
uint16 msgId
uint16 msgVersion
uint32 payloadSize
uint16 spare2


